tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1350,6 +1350,218 @@ Feature: check share activity
+                              "required": ["id","name"],
+                              "properties": {
+                                "id": {
+                                  "type": "string",
+                                  "pattern": "^%file_id_pattern%$"
+                                },
+                                "name": {
+                                  "const": "folderToShare"
+                                }
+                              }
+                            },
+                            "sharee": {
+                              "type": "object",
+                              "required": ["id","displayName"],
+                              "properties": {
+                                "id": {
+                                  "type": "string",
+                                  "pattern": "^%user_id_pattern%$"
+                                },
+                                "displayName": {
+                                  "const": "Brian"
+                                }
+                              }
+                            },
+                            "user": {
+                              "type": "object",
+                              "required": ["id","displayName"],
+                              "properties": {
+                                "id": {
+                                  "type": "string",
+                                  "pattern": "%user_id_pattern%"
+                                },
+                                "displayName": {
+                                  "const": "Alice Hansen"
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                },
+                {
+                  "type": "object",
+                  "required": ["id","template","times"],
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "pattern": "^%user_id_pattern%$"
+                    },
+                    "template": {
+                      "type": "object",
+                      "required": ["message","variables"],
+                      "properties": {
+                        "message": {
+                          "const": "{user} updated {field} for the {resource}"
+                        },
+                        "variables": {
+                          "type": "object",
+                          "required": ["field","folder","resource","user"],
+                          "properties": {
+                            "field": {
+                              "type": "object",
+                              "required": ["id","name"],
+                              "properties": {
+                                "name": {
+                                  "const": "expiration date, permission"
```

Comment:
I do not see role change. does role change have an event?

Reply:
The response message for both expiration date and permission role is given at once:
![image](https://github.com/user-attachments/assets/9f1f47c1-a52a-4d9a-8ca8-d48002d0f678)

Reply:
please, check if the issue has been fixed or not
https://github.com/owncloud/ocis/issues/10228

Reply:
@saw-jan 
Yes, it's been fixed.
Previously, the response message would be:
`Admin updated some field for the f1`
Now, it's
`Admin updated permission for the f1`

Reply:
Ahh, I was thinking of permission and role as different things. my brain